File: 3334-apple-redistribution-into-boxes/apple-redistribution-into-boxes.java
class Solution {
    public int minimumBoxes(int[] a, int[] c) {
        int sum=0;
        for(int i=0;i<a.length;i++){
            sum+=a[i];
        }
        Arrays.sort(c);
        int sum1=0;
        int v=0;
        for(int i=c.length-1;i>=0;i--){
            sum1+=c[i];
            v++;
            if(sum1>=sum){
                return v;
            }
        }
        return 0;
    }
}

File: 3334-apple-redistribution-into-boxes/apple-redistribution-into-boxes.py
class Solution:
    def minimumBoxes(self, a: List[int], c: List[int]) -> int:
        c.sort()
        sum=0
        for i in range(len(a)):
            sum=sum+a[i]
        sum1=0
        v=0
        for i in range(len(c)-1,-1,-1):
            sum1=sum1+c[i]
            v=v+1
            if(sum1>=sum):
               return v
        return 0
